e2e/playwright/tests/visual/intro_channel.spec.ts

```diff
@@ -34,7 +34,7 @@ test('Intro to channel as regular user', async ({page, isMobile, browserName, vi
     await expect(channelPage.postTextbox.input).toBeVisible();
 
     // Hide dynamic elements of the page
-    await page.addStyleTag({content: hideTeamHeader + hidePostHeaderTime});
+    await page.addStyleTag({content: hideTeamHeader + hidePostHeaderTime + hideAppBarPlaybookIcon});
```

Comment:
Temporarily hiding playbook icon in the app bar. Having issue when loading the icon via docker setup. Will address separately.

Reply:
Should be fixed by https://github.com/mattermost/mattermost-webapp/pull/12196